packages/fxa-support-panel/src/app/app.controller.ts

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import {
+  Controller,
+  Get,
+  Query,
+  Render,
+  UseGuards,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { MozLoggerService } from 'fxa-shared/nestjs/logger/logger.service';
+
+import { CurrentUser } from '../auth/auth-header.decorator';
+import { AuthHeaderGuard } from '../auth/auth-header.guard';
+import { RemoteLookupService } from '../remote-lookup/remote-lookup.service';
+import { AccountQuery } from './account-query.dto';
+
+@Controller()
+@UseGuards(AuthHeaderGuard)
+export class AppController {
+  constructor(
+    private log: MozLoggerService,
+    private remote: RemoteLookupService
+  ) {}
+
+  @Get()
+  @UsePipes(new ValidationPipe())
+  @Render('index')
+  async root(@CurrentUser() user: string, @Query() query: AccountQuery) {
```

Comment:
I considered that, but since we're only logging it as a string, it ends up that way anyways. Also, the `ValidationPipe` has a conversion option where it will do the conversion if that's specified.

Reply:
See https://docs.nestjs.com/techniques/validation#transform-payload-objects for some default conversions it can do.